crates/autopilot/src/run.rs

```diff
@@ -334,8 +334,7 @@ pub async fn run(args: Arguments) {
             .unwrap_or_else(|| BalancerFactoryKind::for_chain(chain_id));
         let contracts = BalancerContracts::new(&web3, factories).await.unwrap();
         match BalancerPoolFetcher::new(
-            &args.shared.graph_api_base_url,
-            chain_id,
+            &args.shared.graph_api_url,
```

Comment:
Since `graph_api_url` now has to contain the entire URL (including the specific subgraph name) we'd now have to have 2 separate args for the `balancerv2` and `uniswapv3` subgraph. Otherwise we can't configure both of them together.